src/core_ocean/shared/mpas_ocn_vmix_cvmix.F

```diff
@@ -685,7 +687,7 @@ subroutine ocn_vmix_coefs_cvmix_build(meshPool, statePool, forcingPool, err, tim
          if ( kpp_stage == 2) then
          ! call convective mixing scheme
          if (config_use_cvmix_convection) then
-            do k = 1, maxLevelCell(iCell) + 1
+            do k = minLevelCell(iCell), maxLevelCell(iCell) + 1
```

Comment:
similar comment to above.  I think you'll want to keep this as 1 to zero out the entire array.